### DISCUSSION TITLE
SN dev chat

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#1059] [General]
@stackernews/foss 

We have a self-hosted dev chat running. If you've made contributions and would like to join, drop your SN nym in a reply and I'll send you a lnpay message on SN with instructions. 

(I'm hoping to streamline this signup process in the future but it'll be a little hokey/manual for a minute.)

**Comment by SatsAllDay** [comment]
https://stacker.news/WeAreAllSatoshi

**Reply by huumn** [reply]
Sent!

**Comment by felipebueno** [comment]
https://stacker.news/felipe

**Reply by huumn** [reply]
Sent!

**Comment by dillon-co** [comment]
https://stacker.news/dillon

**Reply by huumn** [reply]
Sent!

**Comment by huumn** [comment]
@benalleng please consider joining the group chat. The bolt11 you gave me before was not settling and you've earned more than I can send to your SN account.

**Reply by benalleng** [reply]
https://stacker.news/BenAllenG sorry! just got a little caught up with the day job today

**Reply by huumn** [reply]
Sent!

**Comment by SouthKoreaLN** [comment]
south_korea_ln

**Reply by huumn** [reply]
Sent!

**Comment by aniskhalfallah** [comment]
little late to the party : 
https://stacker.news/aniskhalfallah

**Reply by huumn** [reply]
Sent to your SN account. Check you notifications.

(Sorry it took so long!)

**Reply by aniskhalfallah** [reply]
Thanks, No worries !
I have a small issue though, I get "Unable to decrypt message" for all chat history, Have you ever encountered that?

**Reply by huumn** [reply]
I don't have that issue because I've been in the chat since the beginning. Maybe ask the question in chat?

**Comment by tsmith123** [comment]
https://stacker.news/t0m

**Reply by huumn** [reply]
Welcome! Sent to your SN account. Check you notifications.

**Comment by riccardobl** [comment]
Hello, i am working on this: https://github.com/stackernews/stacker.news/issues/1142
May i join the dev chat?  https://stacker.news/rblb

**Reply by huumn** [reply]
Chat creds sent to your SN account. Check you notifications.

**Comment by handsomelatino** [comment]
_Edit: SN nym -> https://stacker.news/handsome_latino_

Hi. May I also join the chat?

Edit # 10: Keeping this for posterity, but seems like I also had the /app error. Adding what git recommended on the command line did the trick:

```
git config --global --add safe.directory /app
```

I have absolutely no clue what it does but worked: https://github.com/user-attachments/assets/5efd0545-3c57-4ad2-b976-3f6023202bd8

I get an error when running `./sndev start`:

```
dependency failed to start: container for service "app" is unhealthy
```

Everything got downloaded, compiled, and built; bitcoin, lnd etc. on the console. Most services work, DB, creating ./sndev login, but localhost:3000 hangs

On a note, I had to document all the lines having a `restart` property on `docker-compose.yml` because I'm running an older version of macOS (11) and it won't let me update :(

```
Docker Client Version:           20.10.22
Docker Compose version v2.15.1
```

**Reply by ekzyis** [reply]
Mhh, not sure why this error continues to happen. IIRC the error is triggered by the git call in next.config.js if directory permissions are wrong. But sounds like you resolved it and don't need any further help?

@huumn will respond when he sent you the chat credentials on SN.

**Comment by thebullishbitcoiner** [comment]
https://stacker.news/thebullishbitcoiner

**Reply by thebullishbitcoiner** [reply]
Yo @huumn I got the lnpay message but I'm getting a server can't be found issue when going to https://sndev.chat

**Reply by huumn** [reply]
my bad wrong url https://chat.sndev.team

**Reply by huumn** [reply]
hadn't had my coffee yet

**Reply by thebullishbitcoiner** [reply]
No prob. I'm in! Is there an existing room to join?

**Reply by huumn** [reply]
Yep, just added you (one of those days).

**Comment by toyota-corolla0** [comment]
https://stacker.news/toyota_corolla0

**Reply by huumn** [reply]
Remind me tomorrow if I forget.

**Reply by toyota-corolla0** [reply]
reminder

**Reply by huumn** [reply]
Sorry about my slowness. My life is a mess. 

Please check your SN notifications for login info.

**Reply by toyota-corolla0** [reply]
I am not able to login with the credentials I received.

**Reply by huumn** [reply]
I think I gave you the wrong name. Try `toyota-corolla0` rather than `toyota_corolla0`

**Comment by AndreaDiazCorreia** [comment]
I'm new but a will start to colaborate with you https://stacker.news/andreadcorreia

**Reply by huumn** [reply]
Cool! I'll be happy to add you once you've made some contributions first.

**Reply by AndreaDiazCorreia** [reply]
How could I ask questions? Do you have any other chat options?

**Reply by huumn** [reply]
You can create discussions here or ask the questions on issues? Are you unable to do that? I believe permissions should allow you to.

**Comment by felipebueno** [comment]
Hey @huumn 

I just tried to login on the sndev.team chat but it didn't work. It's been a few months since my last login

![image](https://github.com/user-attachments/assets/793750d0-7893-4d99-a0fe-05f66dff5724)

![image](https://github.com/user-attachments/assets/0f6cc439-adab-43c8-acae-80a9cc539842)

**Reply by ekzyis** [reply]
Hey, I just tried to login with my own credentials and it worked. Are you sure your credentials are correct? I saw the same warning about the identity server but it still worked.

**Reply by felipebueno** [reply]
> Are you sure your credentials are correct?

oops :sweat_smile: 

For some reason, I had two saved credentials for the sndev chat. I tried the second one and it worked

Thanks, @ekzyis

**Comment by Soxasora** [comment]
Could I join too? :P it would help me with decision making on some issues
https://stacker.news/Soxasora

**Reply by huumn** [reply]
Chat creds sent to your SN account! Check you notifications.

**Reply by Soxasora** [reply]
Thanks!!

**Comment by cyphercosmo** [comment]
Please invite me https://stacker.news/cyphercosmo

**Reply by huumn** [reply]
Once you get an issue merged, I'll be happy to!

**Comment by ed-kung** [comment]
Can I be added?

https://stacker.news/SimpleStacker

**Comment by Scroogey-SN** [comment]
https://stacker.news/Scroogey

**Reply by huumn** [reply]
Scroogey did I add you ever?

**Reply by Scroogey-SN** [reply]
No, I don't think so, re-checked the invoices in satistics but didn't see it.

**Comment by cyphercosmo** [comment]
Please add me @huumn

**Comment by brymut** [comment]
Can I be added @huumn https://stacker.news/brymut

**Reply by huumn** [reply]
Sent credentials. Sorry for the delay! 

If you have turned off the payment proxy, you won't be notified and will need to look in `satistics` for the invoice comment.

**Reply by brymut** [reply]
Thanks for the follow up, got in ok 👍

**Comment by sutt** [comment]
Enters the chat...
https://stacker.news/bounty_hunter

**Reply by huumn** [reply]
Sent credentials.

If you have turned off the payment proxy, you won't be notified and will need to look in satistics for the invoice comment.

**Reply by sutt** [reply]
> Sent credentials.
> If you have turned off the payment proxy, you won't be notified and will need to look in satistics for the invoice comment.

Thank you I'm in. Message came right into notifications for me.

**Comment by pory-gone** [comment]
@huumn 
could I come in too? I'd love it!
https://stacker.news/pory_gone

**Reply by huumn** [reply]
Sorry for the delay

I am unabled to send credentials via lnurl to pory@porygone.xyz

please inform me how to get them to you

**Reply by pory-gone** [reply]
Absolutely no problem!

In fact, I apologize for causing so many problems with my node.
But now I've opened it again and everything should be working. Alternatively, I'm subscribed to the Stacker News site and if there's a way, you can send them to me there.

**Reply by huumn** [reply]
Still getting `Recipient wallet error. Please contact the recipient.`

**Reply by pory-gone** [reply]
Mmh Just give me a moment

Ok it was offline but now is up for sure, i'm keeping it under control

**Reply by huumn** [reply]
done

**Reply by pory-gone** [reply]
Thank you so much indeed and sorry again for all the problems!

**Comment by securitybrahh** [comment]
I am trying to find a issue or a PR i can work on. I don't know if I am welcome yet. but here it goes:

https://stacker.news/security_brahh

**Reply by securitybrahh** [reply]
@huumn?

**Reply by huumn** [reply]
We gate the dev chat with proof of work. Consider finding an issue or creating one you'd like to work on.

Please wait for confirmation of our desire to include anything you suggest though. We've had to get rather selective with all the slop rising, and will not add things to the app just cuz you implemented them.
